coriolis/wsman.py

```diff
@@ -86,9 +88,16 @@ def from_connection_info(cls, connection_info):
     def disconnect(self):
         self._protocol = None
 
+    def set_timeout(self, timeout):
+        if timeout > 0:
+            self._protocol.transport.timeout = timeout
```

Comment:
Has this been tested and works? (I'm asking because in the `protocol.Protocol` constructor the `transport` is merely a string)

Reply:
Tested and works. Only acceptable values are ints bigger than 0. No way to disable the timeout.